File: DSA_practice/Prime.java
import java.util.Scanner;
public class Prime{
    public static void main(String[] args) {
        int n=10; 
        int i=2;
        while(i<n){
            if(n%i==0) {
                System.out.println("Not a prime number");
                break;
            }
            i++;
        }
        System.out.println("Prime number");
    }
}

File: DSA_practice/prime.c
#include<stdio.h>
int main(){
    int n=10; 
        int i=2;
        int count=0;
        while(i<n){
            if(n%i==0) {
                count++;
                break;
            }
            i++;
        }
        if(count==0) printf("Prime number");
        else printf("Not a prime number");
    return 0;
}